util/deephash/deephash.go

```diff
@@ -136,7 +137,21 @@ func print(w *bufio.Writer, v reflect.Value, visited map[uintptr]bool, scratch [
 		return acyclic
 	case reflect.Slice, reflect.Array:
 		if v.Type().Elem().Kind() == reflect.Uint8 && v.CanInterface() {
-			fmt.Fprintf(w, "%q", v.Interface())
+			if v.CanAddr() {
+				Len := v.Len()
+				w.Write(strconv.AppendInt(scratch[:0], int64(Len), 10))
+				w.WriteByte(':')
+				if Len > 0 {
+					var s []byte
+					sh := (*reflect.SliceHeader)(unsafe.Pointer(&s))
```

Comment:
this is `unsafe.Slice` in the latest Go?

Reply:
Yes. But can't use it quite yet.